internal/app/machined/pkg/runtime/v1alpha1/platform/nocloud/testdata/metadata-v2.yaml

```diff
@@ -12,6 +12,19 @@ ethernets:
     nameservers:
       search: [foo.local, bar.local]
       addresses: [8.8.8.8]
+  eth1:
+    match:
+      macaddress: '00:20:6e:1f:f9:a9'
+    addresses:
+      - 10.22.14.2/32
+    nameservers:
+      search: [ foo.local, bar.local ]
+      addresses: [ 8.8.8.8 ]
+    routes:
+      - to: "0.0.0.0/0"
+        via: "192.168.14.1"
+        metric: 100
```

Comment:
this is not being used?

Reply:
Oops - good catch thank you! I did most of this work awhile ago before this file got split into metadata-v2-nocloud and metadata-v2-cloudinit

Reply:
I added metric support